homeassistant/components/accuweather/const.py

```diff
@@ -61,145 +63,145 @@
     ATTR_CONDITION_WINDY: [32],
 }
 
-FORECAST_DAYS = [0, 1, 2, 3, 4]
+FORECAST_DAYS: Final[list[int]] = [0, 1, 2, 3, 4]
```

Comment:
Maybe replace this with just an `int`?
```suggestion
MAX_FORECAST_DAYS: Final = 4
```

And then in `sensor.py`:
```python
for day in range(MAX_FORECAST_DAYS + 1):
```

Reply:
Good idea.